src/views/Home/index.view.tsx

```diff
@@ -12,6 +12,7 @@ import SpeakersComponent from "./components/Speakers/index.view";
 import FAQComponent from "./components/FAQ/index.view";
 import SponsorsComponent from "./components/Sponsors/index.view";
 import FooterComponent from "./components/Footer/index.view";
+import Button from "../../components/Button/index.view";
```

Comment:
Because our root folder is `src` we can reference everything from there. This can simply be: 
```suggestion
import Button from "./components/Button/index.view";
```